### ISSUE TITLE
Can be used in a dango view

### ISSUE THREAD

**Issue by teury** [raagin/django-streamfield#38]
It is just what I was looking for, but I see that it is only for the django manager, I would like to use it in a django view, or be able to add elements without the manager, I have not seen documentation on this. thank you so much.

**Comment by raagin** [comment]
Hi! If i understood right, уou have a question about how to use django-streamfield outside django admin panel. Am i right?

**Comment by teury** [comment]
> Hi! If i understood right, уou have a question about how to use django-streamfield outside django admin panel. Am i right?

Yes, I wanted to know if the StreamFieldWidget can work with django forms without the Administration System.

**Comment by raagin** [comment]
Yes, it works. But it doesn't have css styles.
I will care about it in future releases.
You may use standard django modelform for it
```python
# forms.py
from django import forms
from .models import Page

class PageForm(forms.ModelForm):
    class Meta:
        model = Page
        fields = ('title', 'stream',)
```
See more model forms [here](https://docs.djangoproject.com/en/4.0/topics/forms/modelforms/)

Your html:
```html
# page_edit.html
<form method="POST">{% csrf_token %}
{{ form.as_p }}
<div style="clear: both">
<input type="submit">Save</button>
</div>
</form>
<script>
    const django = {jQuery: $}
</script>
{{ form.media }}
```
Important: You need to set in settings.py ```STREAMFIELD_DELETE_BLOCKS_FROM_DB = False```
Or add jQuery on your page and assign it to django object like this ```const django = {jQuery: $}```
Don't forget ```form.media```
